docs/data/sdks/react-native/index.md

```diff
@@ -9,10 +9,13 @@ icon: fontawesome/brands/react
 
 This is the official documentation for the Amplitude Analytics React Native SDK.
 
+!!!info "Legacy SDK"
+    This SDK is legacy and only continue to receive bug fixes until deprecated. Checkout and upgrade to the [React Native TS SDK](../react-native-sdk) which supports plugins and more.
+
 !!!info "SDK Resources"
-    - [React SDK Reference :material-book:](https://amplitude.github.io/Amplitude-ReactNative/)
-    - [React SDK Repository :material-github:](https://github.com/amplitude/Amplitude-ReactNative)
-    - [React SDK Releases :material-code-tags-check:](https://github.com/amplitude/Amplitude-ReactNative/releases)
+    - [React Native SDK Reference :material-book:](https://amplitude.github.io/Amplitude-ReactNative/)
+    - [React Native SDK Repository :material-github:](https://github.com/amplitude/Amplitude-ReactNative)
+    - [React Native SDK Releases :material-code-tags-check:](https://github.com/amplitude/Amplitude-ReactNative/releases)
 
 !!!info "New Analytics SDK for React Native Available in Beta"
```

Comment:
Use custom "Beta" admonition here instead of "info"? 
<img width="53" alt="image" src="https://user-images.githubusercontent.com/5790872/185003099-7d08bf38-9c24-48f6-b9e2-bde429cffaca.png">

Reply:
This block looks like a new SDK up sell. I merged it with legacy warning and up sell.